cli/ballerina-launcher/src/main/resources/cli-help/ballerina-init.help

```diff
@@ -12,8 +12,11 @@ DESCRIPTION
      sample Ballerina source file. You can use Ballerina.toml to manage
      the organization setting and version for the project.
      
-     The generated set of artifacts also includes a service skeleton 
-     and a test skeleton that you can use to kick start the project. 
+     If you run the init command in the interactive mode (--i) and
```

Comment:
--i  should be -i

Reply:
We also have the -interactive flag right ?

Reply:
Changed --i to -i and added the --interactive flag with 1cdb231